### PR TITLE
Fix for iOS and OSX

### PR DESCRIPTION
**Pull request by ErickXavier**
Replaces DragEvent for any, to work properly in iOS and OSX devices

### CODE REVIEW

File: src/lib/directives/drop-zone/drop-zone.directive.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class DropZoneDirective {
      */
     @HostListener('drop', ['$event'])
     public onDrop(event: any): void {
-        if (!this.disabled && event instanceof DragEvent) {
+        if (!this.disabled && event instanceof any) {
```
Comment (troymjones) [inline src/lib/directives/drop-zone/drop-zone.directive.ts]: This won't compile/work. I think the `event instanceof` check can just be removed.

**Comment by troymjones** [comment]
It tried this and it works on macOS 10.4.6 - Safari 12.1.2

File: src/lib/directives/drop-zone/drop-zone.directive.ts (hunk 1 above)
Comment (rsaenen) [reply src/lib/directives/drop-zone/drop-zone.directive.ts]: Thanks, please remove the instanceof any before I merge it.

Comment (ErickXavier) [reply src/lib/directives/drop-zone/drop-zone.directive.ts]: Done.

**Comment by ErickXavier** [comment]
> Remove `event instance of any`

Done.

**Pull request merged**